ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/result/impl/PreventiveAndCurativesRaoResultImpl.java

```diff
@@ -359,6 +367,17 @@ private double getHighestFunctionalForInstant(Instant instant) {
         return highestFunctionalCost;
     }
 
+    private double getTotalFunctionalCostForInstant(Instant instant) {
+        double preventiveCost = secondPreventivePerimeterResult.getFunctionalCost();
+        return preventiveCost
+            + postContingencyResults.entrySet().stream()
```

Comment:
I am dealing with 2P in #1194 because there is no way to optimize linear range actions cos-wise with this current PR
but the cost is based on activated range actions so I do not think that they will be counted twice

Reply:
it could be worth to add a unit test with this situation in #1194 though